Content guidelines (#75)
userstyles-world/userstyles.world: docs/content-guidelines.md

# Kind Content guidelines

We'd like to clarify that these guidelines are only effective on the
UserStyles.world website.

## Purpose

UserStyles.world("USw") encourages their users from anyone who wishes to
add/import/review/explore UserCSS styles("styles"), to be able to do so.
People shouldn't feel discouraged from participating in this community.

In order to ensure the safety and quality of styles, we encourage anyone
to make a conscious effort into following these guidelines
to the best of their ability.

> We'd like to note and clarify that measurements[¹](#footnote) can be taken

Reviewer: measurements should be measures

Author: "measures" seems to be correct as well in this form. However I'm not sure if someone with basic English knowledge understand it the way as intended, as we don't like to use confusing wording. I myself had to check up some dictionary's to see the correct form of this. @vednoc

Reviewer 2: Perhaps "actions" would be better, in which case "can" should be "will". We'll be taking actions if one or more guidelines are broken.

Reviewer: agreed, that sounds better and is more clear as well

Author: All good now. The footnote explains that ^ when actions will be taken.

to ensure the safety and quality of the USw platform.

These guidelines suggest specific ways to accomplish that goal.

## Guidelines

- As a platform that hosts UserCSS styles, USw highly discourages to host/post
    other content then UserCSS styles. 

- USw allows hosting screenshot(s) of your style(s), please try to reflect
    this  screenshot to show how your style looks like when applied.

- As noted, USw hosts screenshot(s) for their styles, please remember that not
    everyone is of consent age in their respective country, therefore
    we discourage the usage of not safe for work ("NSFW") images
    for these screenshot(s).

> - Please, while you're writing a review of a style to give constructive

Reviewer: that's not a valid sentence - I think better would be:

> Please, when you're writing a review of a style, give *constructive* feedback, which will help the author improve the style.
> USw discourages writing reviews that are based on feedback which doesn't help the author improve the style

    feedback, this will help the author improve the style.
    USw discourages writing reviews when these are based on feedback
    that doesn't help the author.
    
- Please, remember that you don't have to write
    a review if you don't want/have to.

- Please, recognize that criticism/feedback of your styles is not
    a personal attack on you.

- Please, don't raise unrelated topics/off-topic while writing a review.

- Please, don't use the review/feedback system to report bugs. While these help
    the author, this system is not the correct way to report them. Try to 
    contact the author and check for a support URL in the UserCSS style.

- Please don't upload styles that discriminate against certain groups of people

> - Please, don't try to post other people's content, but if you do ensure that

Reviewer: some commas would be better this way:
> Please don't try to post other people's content, but if you do, ensure that the license of the original content allows this.

Slight grammar-related edit:
> USw highly discourages posting other people's work if their license is not permissive of it.
> Also, please, try to give credit to the original author.

    the license of the original content allows this. USw highly discourages
    posting other people's work their license is not permissive.
    Also, please, try to give credits to the original author.

## Footnote

> 1. To clarify "Measurements", we say that trusted users and admins have the

Reviewer: "measures" here as well

# Kind Content guidelines

We'd like to clarify that these guidelines are only effective on the
UserStyles.world website.

## Purpose

UserStyles.world("USw") encourages their users from anyone who wishes to
add/import/review/explore UserCSS styles("styles"), to be able to do so.
People shouldn't feel discouraged from participating in this community.

In order to ensure the safety and quality of styles, we encourage anyone
to make a conscious effort into following these guidelines

> best to their ability.

Reviewer 2: ```suggestion
to the best of their ability.
```

We'd like to note and clarify that measurements[¹](#footnote) can be taken
to ensure safety and quality of the USw platform.

These guidelines suggest specific ways to accomplish that goal.

## Guidelines

> - As a platform that host UserCSS styles, USw highly discourage to host/post

Reviewer 2: ```suggestion
- As a platform that hosts UserCSS styles, USw highly discourages to host/post
```

    other content then UserCSS styles. 

> - USw allow to host screenshot(s) for your style, please try to reflect this 

Reviewer 2: ```suggestion
- USw allows hosting screenshot(s) of your style(s), please try to reflect this 
```

>     screenshot to show how your style looks like.

Reviewer 2: ```suggestion
    screenshot to show how your style looks like when applied.
```

- As noted, USw hosts screenshot(s) for their styles, please remember that not

>     everyone is of consent age in their respective country, therefor

Reviewer 2: ```suggestion
    everyone is of consent age in their respective country, therefore
```

>     we discourage the usage of not safe for work("NSFW") pictures

Reviewer 2: ```suggestion
    we discourage the usage of not safe for work ("NSFW") images
```

    for these screenshot(s).

- Please, while you're writing a review of a style to give
    constructive feedback, this will help the author improve the style.

>     USw discourage to write reviews when these are based on feedback

Reviewer 2: ```suggestion
    USw discourages writing reviews when these are based on feedback
```

    that doesn't help the author.
    
- Please, remember that you don't have to write
    a review if you don't want/have to.

- Please, recognize that criticism/feedback of your styles is not a personal attack
    on you.

- Please, don't raise unrelated topics/off-topic while writing a review.

> - Please, don't use the review/feedback system to report bugs, while these help

Reviewer 2: ```suggestion
- Please, don't use the review/feedback system to report bugs. While these help
```

>     the author. This system is not the correct way to report them. Try to to

Reviewer 2: ```suggestion
    the author, this system is not the correct way to report them. Try to to
```

    contact the author and check for a supportURL in the UserCSS style.

- Please, try to avoid offend on gender, race, ethnic group,
    physical appearance, religion, cultural background, and any other
    demographic characteristics views
    to be reflected on UserCSS Styles, USw highly discourages these styles,
    as they can offend people in that group.
    Supporting any of those views is fine to be reflected on UserCSS styles,

>     as long as they don't offend that view.

Reviewer 2: Kinda confusing. Try tidying it up.

Author: Heh, I couldn't. If you get what I'm trying to say. Please try to improve it.

Author: So let's take 2 examples, which both should be covered by this guideline.

Because of 🏳️‍⚧️ 🏳️‍🌈 , which is covered under gender. We are taking it as a example.

1) Let's say Style 1 is going to color youtube.com into a more pride flag-ish background-color. That is perfectly fine because that would support the view.

2) Let's say Style 2 is going to try to undermine the pride and color certain flags like poop color and or things you can imagine which clearly indicate to offend that view.

Reviewer 2: The "view" is what makes it confusing. This is essentially saying to avoid making styles that would ridicule, paint in a negative light, or attack people based on things that they're a part of.

Reviewer: Question: why should userstyles.world care about politcal views? I think the best thing to do is to not open that can of worms imo.

Reviewer 2: I'm not big on politics, but we need to set a boundary on what's acceptable and what isn't. Hosting hate userstyles on our site is not something that I would like to see.

Reviewer: Fair enough, but I'd change the wording to "Please don't upload styles that discriminate against certain groups of people." - I think that's much more compact and clear, doesn't leave anything important out (imo) and still avoids taking a stance on any specific issue.

Reviewer 2: I agree, it sounds great.

Author: Thank you for your suggestion, seems to perfectly word it what was trying to describe. It has been applied.

> - Please, don't try to post others people content, but if you do ensure that

Reviewer 2: ```suggestion
- Please, don't try to post other people's content, but if you do ensure that
```

    the license of the original content allows this. USw highly discourages

>     to post others people content when this isn't allowed. Also please, try to

Reviewer 2: ```suggestion
    posting other people's work their license is not permissive. Also, please, try to
```

>     give credits to the original work.

Reviewer 2: ```suggestion
    give credits to the original author.
```

## Footnote

1. To clarify "Measurements", we say that trusted users and admins have the

>    ability to remove styles/users if they heavily and/or repeatly violate

Reviewer 2: ```suggestion
   ability to remove styles/users if they heavily and/or repeatedly violate
```
